docs/_docs/04_reference/cli-options.md

```diff
@@ -564,7 +565,7 @@ When passed in, don't automatically retry commands which fail with transient err
 ### terragrunt-non-interactive
 
 **CLI Arg**: `--terragrunt-non-interactive`<br/>
-**Environment Variable**: `TF_INPUT` (set to `false`)
+**Environment Variable**: `TERRAGRUNT_NON_INTERACTIVE` (set to `true`)
```

Comment:
Was this a fix for a copy/paste error? Or was this renamed?

Reply:
At first I thought it was a copy/paste error (not mine), but now I realized this is Terraform env var [TF_INPUT](https://developer.hashicorp.com/terraform/cli/config/environment-variables#tf_input)

As I just understood, as it was originally intended, if we disable terraform input by this env var, then this should also enable non-interactive mode in Terragrant
![Code search results · GitHub 2023-07-12 at 3 22 39 PM](https://github.com/gruntwork-io/terragrunt/assets/62853952/7f4327cf-db2f-46a2-bd6b-15c0db22f2af)

If yes, then it seems to me that it looks a bit unusual, doesn't it? Anyway, should I return this logic?

Reply:
Ah, good find. We should maintain backward compatibility with the `TF_INPUT` env var. We could additionally add a TG-specific env var for this as you have, but I wouldn't remove the `TF_INPUT` behavior or the mention in the docs. If anything, might be worth adding in the docs that `TF_INPUT` is native to TF!

Reply:
Agree! We should be ensure, that these improvements will not harm the workflow of customers who were using an older version. Kept backwards compatibility. Updated the docs.